apps/api/project.json

```diff
@@ -15,15 +15,9 @@
       },
       "configurations": {
         "production": {
-          "optimization": true,
+          "optimization": false,
```

Comment:
`"optimization": true` seems to be breaking the API code somehow. It shouldn't be needed anyway since all it does is minimize the code.